common.sh

```diff
@@ -9,6 +9,7 @@
 FUZZING_ENGINE=${FUZZING_ENGINE:-"libfuzzer"}
 [[ $FUZZING_ENGINE != "libfuzzer" ]] && [[ $FUZZING_ENGINE != "afl" ]] && echo "USAGE: If defined, $FUZZING_ENGINE should be either 'afl' or 'libfuzzer' but it is $FUZZING_ENGINE" && exit 1
 
+BINARY_NAME_EXT="_${FUZZING_ENGINE}"
```

Comment:
This will make libFuzzer-based binaries have a different name, which is undesirable, at least at this point, as it will require changes in tutorial.libfuzzer.info. 

Let's not mix changes in the harness and adding a new benchmark.